src/jshint.js

```diff
@@ -904,7 +900,8 @@ var JSHINT = (function() {
         error("E030", state.tokens.curr, state.tokens.curr.id);
       }
 
-      while (rbp < state.tokens.next.lbp && !isEndOfExpr()) {
+      while ((rbp < state.tokens.next.lbp || state.tokens.next.type === "(template)") &&
```

Comment:
On Thu Feb 26 2015 at 12:17:05 PM ⭐caitp⭐ <notifications@github.com> wrote:

> In src/jshint.js
> <https://github.com/jshint/jshint/pull/2196#discussion_r25445080>:
>
> > @@ -904,7 +900,8 @@ var JSHINT = (function() {
> >          error("E030", state.tokens.curr, state.tokens.curr.id);
> >        }
> >
> > -      while (rbp < state.tokens.next.lbp && !isEndOfExpr()) {
> > +      while ((rbp < state.tokens.next.lbp || state.tokens.next.type === "(template)") &&
>
> it's a TODO, i'm not sure how to make it fit into the pratt mechanics.
> I'll just add a comment marking it as such.
>
Got it.

>